CONTRIBUTING.rst

```diff
@@ -28,6 +28,32 @@ request" instead (or waiting to request reviewers).
 .. [*] You'd think this would be obvious, but far too many people ignore
        ``README``\ s in practice
 
+The TL;DR of our review process
+===============================
+
+A detailed description can be found in a separate document about the `review
+process <https://cosmo-epfl.github.io/librascal/dev_guide/review_process.html>`_
```

Comment:
Ah, I didn't see this before. Yes, this looks like the right way to link _to_ the online HTML docs.